pkg/virt-config/config_test.go

```diff
@@ -482,4 +482,38 @@ var _ = Describe("ConfigMap", func() {
 		emulation = clusterConfig.IsUseEmulation()
 		Expect(emulation).To(BeFalse())
 	})
+
+	It("when feature-gate SRIOVLiveMigration is on, LiveMigrationEnabled should return true", func() {
```

Comment:
I think this fits a table test.
You pass the feature-gates in with the expected results of the two.

This seems a classic case where it fits.

Reply:
done